packages/html-reporter/src/testCaseView.tsx

```diff
@@ -14,6 +14,7 @@
   limitations under the License.
 */
 
+import { CopyToClipboard } from '@web/components/copyToClipboard';
```

Comment:
Oh-ah, sorry for not spotting this earlier. I thought the change modifies trace viewer, but in fact it changes the html report. Trace Viewer uses VS Code design language (@web), while HTML Reporter uses GitHub design language, does not depend on @web. So if you are interested in the change to the html report, you should use svg for images, similarly to everything else in the HTML report code.

Reply:
Ah, didn't know that two different design languages where used. I thought @web was a "common area".
I'll close this PR and open a new one since I will need to undo most of the changes.